pkgs/development/tools/out-of-tree/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, buildGoPackage, fetchgit, qemu, docker }:
+
+buildGoPackage rec {
+  name = "out-of-tree-${version}";
```

```suggestion
  pname = "out-of-tree";
```

pname and version get joined automatically as `name = "${pname}-${version}"`

---

fixed